SConstruct

```diff
@@ -1496,10 +1499,8 @@ if testEnv["PLATFORM"] == "darwin" :
 def makeSymLinks( env, target ) :
 
 	links = {
-		"$IECORE_MAJORMINORPATCH_VERSION" : "$IECORE_MAJORMINOR_VERSION",
-		"$IECORE_MAJORMINOR_VERSION" : "$IECORE_MAJOR_VERSION",
-		"${IECORE_MAJORMINORPATCH_VERSION}" : "${IECORE_MAJORMINOR_VERSION}",
-		"${IECORE_MAJORMINOR_VERSION}" : "${IECORE_MAJOR_VERSION}",
+		"${IECORE_VERSION}" : "${IECORE_COMPATIBILITY_VERSION}",
```

Comment:
We seem to have dropped the MAJOR.MINOR -> MAJOR.MINOR.PATCH symlink, but that seems like a reasonable thing to want still (well, MILESTONE.MAJOR.MINOR -> MILESTONE.MAJOR.MINOR.PATCH).

Reply:
Happy to add that one, I just don't have a current use case for it

Reply:
Addressed in 4382448